modules/bridge/src/main/ts/ephox/bridge/components/toolbar/ToolbarButton.ts

```diff
@@ -27,7 +27,7 @@ export interface ToolbarButtonInstanceApi extends BaseToolbarButtonInstanceApi {
 }
 
 export interface BaseToolbarButton<I extends BaseToolbarButtonInstanceApi> {
-  disabled: boolean;
+  enabled: boolean;
```

Comment:
Was this deprecated previously? I'm just wondering if we should leave the old `disabled` flag in place until 7.

Reply:
There is no mention about whether it was deprecated. This will need a medium effort to QA, not sure it will be ready before code freeze

Reply:
That's fine, we can wait until we hear if it was deprecated.

My concern is if it hasn't been pre-announced this will be quite a disruptive change (particularly for customers not using TypeScript, where they will get no error just buttons that aren't disabled correctly).

Reply:
This is far less invasive than some other changes (e.g. take Millie's toolbar/menu renaming PR from earlier this week). Also remember that if renaming it doesn't need to be deprecated, as the functionality isn't being removed. functions do we keep them as well? Then there's also the skin changes where it's pretty evident we're pretty happy with breaking things/making things where the integrator is going to have to go make changes anyways.

That said I'm not against keeping it as a deprecated property if we want, but where do we draw the line? What about the core and toolbar/menu API or similar changes?

Reply:
As long as it's been considered. That's fine.